### ISSUE TITLE
Formating and Newlines

### ISSUE THREAD

**Issue by stephenbanghart** [CISecurity/ROLIE#16]
The main document current suffers from some formatting issues mostly regarding inconsistent use of spacing, indentation, and newlines. The draft switches between using `<vspace/>,` `<vspace blankLines="1">`, and multiple `<t>` sections. We need to decide/figure out which system we are going to use and normalize the document.

Many lines also need to be fixed to avoid going over the character per line limit, as that is producing xml2rfc errors.

**Comment by adammontville** [comment]
How do you propose moving forward?  Does it make sense to consider an alternate form of editing (i.e. a markdown approach)?

**Comment by stephenbanghart** [comment]
I think removing all vspace and replacing it with t tags for each new paragraph is probably the best option in terms of newlines.

I did some more testing and found that all the lines that are too long are from the new example section, some of the code examples end up too long. I'm not sure how best to deal with that as many of the examples would look strange if we broke them across two lines.
